Integrations/integration-SplunkPy.yml

```diff
@@ -138,6 +138,31 @@ script:
         if len(service.jobs) >= 0:
             demisto.results('ok')
         sys.exit(0)
+    if demisto.command() == 'splunk-search':
+        t = datetime.utcnow() - timedelta(days=7)
+        time = t.strftime("%Y-%m-%dT%H:%M:%S")
+        kwargs_oneshot = {"earliest_time":time}
+        if demisto.get(demisto.args(), 'earliest_time'):
+            kwargs_oneshot['earliest_time'] = demisto.args()['earliest_time']
+        if demisto.get(demisto.args(), 'latest_time'):
+            kwargs_oneshot['latest_time'] = demisto.args()['latest_time']
+        if demisto.get(demisto.args(), 'event_limit'):
+            kwargs_oneshot['count'] = int(demisto.args()['event_limit'])
+        searchquery_oneshot = demisto.args()['query']
+        if not searchquery_oneshot.startswith('search'):
+            searchquery_oneshot = 'search '+searchquery_oneshot
+        oneshotsearch_results = service.jobs.oneshot(searchquery_oneshot, **kwargs_oneshot)
+
+        reader = results.ResultsReader(oneshotsearch_results)
+        res = []
+        for item in reader:
+            if isinstance(item, results.Message):
+                demisto.results({"Type": 1, "ContentsFormat": "json", "Contents": json.dumps(result.message)})
+            elif isinstance(item, dict):
+                # Normal events are returned as dicts
+                res.append(item)
+        demisto.results({"Type": 1, "ContentsFormat": "json", "Contents": json.dumps(res)})
```

Comment:
1. I think we should present it in markdown table if possible. here is the function I used in other python integration
```
def create_entry(header, table, context = {}, headers = None):
    return {
        'ContentsFormat': formats['json'],
        'Type': entryTypes['note'],
        'Contents': table,
        'ReadableContentsFormat': formats['markdown'],
        'HumanReadable': tableToMarkdown(header, table, headers) if table else 'No result were found',
        'EntryContext': context
    }
```
 - looks like every entry data is under `_raw` filed so don't forget to extract it
 2. I think we can add it to context also (under `Splunk.Search` maybe as is)